### ISSUE TITLE
documentation: integration with DAWN or pyFAI

### ISSUE THREAD

**Issue by mikapfl** [LightForm-group/xrdfit#50]
Hi,

I think it would be great if you could provide some pointers (or an example script or such) how to best integrate with DAWN or pyFAI. I don't think you need to cover the whole topic of how to use pyFAI for caking, but maybe provide the correct calls to `pyfai.io` to save the data in a format which can be loaded easily in xrdfit.

For DAWN, I'm not quite sure if the documentation is clear at the moment becaue I haven't used DAWN much, I guess I would have to use the `Export to Text File` function? DAWN has some hdf5 io functions though, but I guess they are not appropriate for consumption with `xrdfit`?

Cheers,

Mika

**Comment by merrygoat** [comment]
Thank you, a great suggestion. @christopher-daniel has done some work on an example script for pyFAI caking. He is just tidying some bits and adding some documentation and should be able to update in a few days.

**Comment by merrygoat** [comment]
@mikapfl Thank you for the suggestion of pyFAI. I just spent the best part of a day diving into it and while it was hard to get my head around, now I have got it sorted its really good and integrates really well with xrdfit as part of a reproducible analysis pipeline. I've added a couple of minor features to make xrdfit more flexible in reading data files. 

We are going to do a proper tutorial/example notebook on the caking for our students to use, so will link that in to the xrdfit tutorials/documentation too.
